### ISSUE TITLE
Consider Adding Kanji Dictionary

### ISSUE THREAD

**Issue by Hapoofesgeli** [luke-c/Ikue#3]
It would be nice to have a Kanji dictionary, so we can also check the Kanjis of searched vocabulary etc. (it would also be nice having handwriting recognition for it, I believe there are already open-source implementions of it around)

**Comment by luke-c** [comment]
Definitely! It is in the pipeline but I'm not going to lie it's going to take a while. Before I start working on any new features there's a lot of refactoring and reworks planned, but after that this is one of the features near the top. 

So no promises on an ETA but it is something that I want to do (eventually).

**Comment by nicolasmaia** [comment]
Still holding out hope on this one.

I tried other dictionaries, but Ikue is still the one I like the most.
